styles/custom-viewer-styles.css

```
@@ -1,7 +1,33 @@
+
+  /* Disable pointer events for saving */
+  pointer-events: auto;
+
+  /* Prevent context menu */
+  -webkit-context-menu: none;
+}
+
+.viewer-image-mobile {
+  -webkit-user-drag: none !important;
+  -webkit-user-select: none !important;
+  -khtml-user-select: none !important;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)